source/conf.py-intl

```diff
@@ -0,0 +1,412 @@
+# -*- coding: utf-8 -*-
```

Comment:
BTW, this is used only for getting the source strings because sphinx-intl currently cannot handle "tag"